src/utils.js

```diff
@@ -11,3 +11,19 @@ export function isEnterpriseLearner(user) {
 
   return false;
 }
+
+// eslint-disable-next-line import/prefer-default-export
+export function getPreferredEnterpriseUUID(user) {
+  if (user !== null && user.roles) {
+    const { roles } = user;
+    for (let i = 0; i < roles.length; i += 1) {
+      if (roles[i].includes('enterprise_learner')) {
```

Comment:
I'm also wondering if these functions should be tied specifically to the `enterprise_learner` role like they are now, or if it should be more flexible to handle all role types: learners, admins, or openedx operators.

Reply:
Oh excellent point, could pass that in as a param and make a set of constants available (if they're not already).

Might punt on this just in the short term because I'd rather things be consistent and I'm not looking to fix the isEnterpriseLearner method and everything downstream that calls it.

Unless you're saying we just change both of these methods to include every enterprise role type?

Reply:
Yeah, I'm suggesting making both function to be more generic to answer the question... "is this user Enterprise-related (any enterprise role)?" and, if so, "give that user their 'preferred' learner portal link".

Mainly suggesting because I've heard some feedback that admins might want to be using the Learner Portal as well (won't just be for learners). And if that's case, the navigation links from edx.org to LP should probably work for all role types.